proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -448,8 +448,8 @@ public static VelocityConfiguration read(Path path) throws IOException {
     CommentedFileConfig defaultConfig = CommentedFileConfig.of(tmpFile, TomlFormat.instance());
     defaultConfig.load();
 
-    // Whether or not this config is version 1.0 which uses the deprecated "forwarding-secret" parameter
-    boolean legacyConfig = config.getOrElse("config-version", "").equalsIgnoreCase("1.0");
+    // Whether or not this config version is older than 2.0 which uses the deprecated "forwarding-secret" parameter
+    boolean legacyConfig = Double.parseDouble(config.getOrElse("config-version", "1.0")) < 2.0;
```

Comment:
Nit: a version is not guaranteed to follow the `major.minor` format. The Polymer branch included the semver4j dependency to perform plugin dependency matching (see [diff](https://github.com/PaperMC/Velocity/commit/2a39ddb03e5a25dafaa3949859e2860ee3650656#diff-f091b0b868fccf7d828aa915340bf2d65c580d71bd5d23e0af8f44fcd1a231c1R138)). While this will work for now, I would add a `// todo` annotation.

Reply:
Currently the config-version option accepts a String, in case it is not a double, the version will be set to 1.0. I also put a TODO to change the logic in Polymer